tests/test_wave_01.py

```diff
@@ -25,14 +25,15 @@ def test_get_tasks_one_saved_tasks(client, one_task):
     assert response_body == [
         {
             "id": 1,
+            "goal_id": None,
```

Comment:
Please do NOT modify tests in places where you are not asked to. This test was written this way on purpose. Tasks should only display `goal_id` if it has a goal id. As we have talked about in class, when we do Test Driven Development, we let the tests guide our implementation. If this is what the tests want, we need to make our code work that way.